porick/lib.py

```diff
@@ -1,12 +1,48 @@
 import bcrypt
 import re
+import smtplib
+
+from email.mime.text import MIMEText
 
 from flask import request
 from sqlalchemy import or_
 
 from models import User
 from porick import db
-from settings import PASSWORD_SALT
+from settings import PASSWORD_SALT, SERVER_DOMAIN, PASSWORD_RESET_REQUEST_EXPIRY, SMTP_REPLYTO, SMTP_SERVER
+
+
+
+reset_password_text = """
```

Comment:
If this is going to live here (in the global namespace), it should be `RESET_PASSWORD_TEXT`.

At the same time, I'd much rather a template, say, `/templates/email/password_reset.txt`. Then, you can simply do `body = render_template('/templates/email/password_reset.txt', domain=domain, key=key, validity=validity)` to render it, and the text of the message is nicely hidden, away from the python module.